dev/tailwind.config.site.js

```diff
@@ -12,23 +12,23 @@ const colors = require('tailwindcss/colors')
 
 module.exports = {
   theme: {
-    // Here we define the default colors available. If you want to include
-    // all default Tailwind colors you should extend the colors instead.
-    colors: {
```

Comment:
I noticed you moved this out of the colors and into extend though, was that on purpose? Extend should extend and not constrain.